[eve]  port can't use after installed.

**Describe the bug**
A clear and concise description of what the bug is.
after ./vcpkg install eve
add this line  find_package(eve CONFIG REQUIRED) to CMakeLists.txt
and when configure , report can't find eve-config.cmake.



**Environment**
- OS: Linux
- Compiler: gcc 11.2

**To Reproduce**
Steps to reproduce the behavior:
1. ./vcpkg install eve
2. add  find_package(eve CONFIG REQUIRED) 
3. configure add see configure error
Repro code when

**Expected behavior**
Should can use find_package to find eve port.

**Failure logs**
-(please attached failure logs)
1> [CMake] CMake Error at /home/rob/vcpkg/scripts/buildsystems/vcpkg.cmake:824 (_find_package):
1> [CMake]   Could not find a package configuration file provided by "eve" with any of
1> [CMake]   the following names:
1> [CMake] 
1> [CMake]     eveConfig.cmake
1> [CMake]     eve-config.cmake
1> [CMake] 
1> [CMake]   Add the installation prefix of "eve" to CMAKE_PREFIX_PATH or set "eve_DIR"
1> [CMake]   to a directory containing one of the above files.  If "eve" provides a
1> [CMake]   separate development package or SDK, be sure it has been installed.
1> [CMake] Call Stack (most recent call first):
1> [CMake]   clang_rename/CMakeLists.txt:8 (find_package)
**Additional context**
Add any other context about the problem here.

--- a/ports/eve/portfile.cmake
+++ b/ports/eve/portfile.cmake
@@ -1,15 +1,29 @@
-message(WARNING "EVE requires a C++ 20 compliant compiler. GCC-11 and clang-12 are known to work.")
+vcpkg_minimum_required(VERSION 2022-10-12) # for ${VERSION}
 
+string(REGEX REPLACE "^(v[0-9]+)[.]([0-9])[.]([0-9]+)\$" "\\1.0\\2.\\3" git_ref "v${VERSION}")
 vcpkg_from_github(
     OUT_SOURCE_PATH SOURCE_PATH
     REPO jfalcou/eve
-    REF v2022.09.0
-    SHA512 ab5be8c897955e08e1aa192ac9dd90e310b2786a2f295b0d5a5d309fa8e621b66673668b9dbe2f683a5e2596d291b0521d80a8bb80f493ecb12e86ab5d830c7b
+    REF "${git_ref}"
+    SHA512 5623587de77c3e321555ca99326829928f69c5ac499d2abedfb18860e8c2a4c7dd5b408afef3ae2f8681cb363ffbfbc869dbf28c90e62e2b0abca62e03d08d12
     HEAD_REF main
 )
+
+set(VCPKG_BUILD_TYPE release) # header-only port
 
 vcpkg_cmake_configure(SOURCE_PATH "${SOURCE_PATH}")
 
 vcpkg_cmake_install()
-file(REMOVE_RECURSE "${CURRENT_PACKAGES_DIR}/debug")
-file(INSTALL "${SOURCE_PATH}/LICENSE.md" DESTINATION "${CURRENT_PACKAGES_DIR}/share/eve" RENAME copyright)
+
+vcpkg_cmake_config_fixup(CONFIG_PATH "lib/eve-${VERSION}")
+if(NOT EXISTS "${CURRENT_PACKAGES_DIR}/share/eve/eve-config.cmake")
+    message(FATAL_ERROR "CMake config is missing")
+endif()
+
+file(REMOVE_RECURSE
+    "${CURRENT_PACKAGES_DIR}/lib"
+    "${CURRENT_PACKAGES_DIR}/share/doc"
+)
+
+file(INSTALL "${CMAKE_CURRENT_LIST_DIR}/usage" DESTINATION "${CURRENT_PACKAGES_DIR}/share/${PORT}")
+file(INSTALL "${SOURCE_PATH}/LICENSE.md" DESTINATION "${CURRENT_PACKAGES_DIR}/share/${PORT}" RENAME copyright)

--- a/ports/eve/usage
+++ b/ports/eve/usage
@@ -1,0 +1,7 @@
+eve provides CMake targets:
+
+    find_package(eve CONFIG REQUIRED)
+    target_link_libraries(main PRIVATE eve::eve)
+
+Using eve requires a C++20 compliant compiler.
+GCC-11 and clang-12 are known to work.

--- a/ports/eve/vcpkg.json
+++ b/ports/eve/vcpkg.json
@@ -1,11 +1,14 @@
 {
   "name": "eve",
-  "version-date": "2022-09-20",
-  "description": "EVE - the Expressive Vector Engine",
+  "version": "2022.9.1",
+  "description": [
+    "EVE - the Expressive Vector Engine",
+    "A header-only C++20 and onward implementation of a type based wrapper around SIMD extensions sets for most current architectures."
+  ],
   "homepage": "https://github.com/jfalcou/eve",
   "documentation": "https://jfalcou.github.io/eve/",
   "license": "BSL-1.0",
-  "supports": "!windows",
+  "supports": "!windows, mingw",
   "dependencies": [
     {
       "name": "vcpkg-cmake",

--- a/versions/baseline.json
+++ b/versions/baseline.json
@@ -2181,7 +2181,7 @@
       "port-version": 1
     },
     "eve": {
-      "baseline": "2022-09-20",
+      "baseline": "2022.9.1",
       "port-version": 0
     },
     "eventpp": {

--- a/versions/e-/eve.json
+++ b/versions/e-/eve.json
@@ -1,6 +1,11 @@
 {
   "versions": [
     {
+      "git-tree": "701ec863d463d1a8c570336d3c5bd672f540793c",
+      "version": "2022.9.1",
+      "port-version": 0
+    },
+    {
       "git-tree": "3b6a47336d9cf0e99ea0dffc7c0c41bb04dfff5a",
       "version-date": "2022-09-20",
       "port-version": 0

